doc/ref_model/chapters/chapter07.md

```diff
@@ -96,6 +98,41 @@ At a high level, the following areas/requirements cover workload security for a
 * Workload owner owns workload design change process
 * Workload owner owns workload software update process
   
+ <a name="7.3.3"></a> 
+ ### 7.3.3 Common Security Standards
+
+The Cloud Infrastructure Reference Model and the supporting architectures are not only required to optimally support networking functions, but they must be designed with common security principles and standards from inception.  These best practices must be applied at all layers of the infrastructure stack and across all points of interconnections with outside networks, APIs and contact points with the NFV network functions overlaying or interacting with that infrastructure.
```

Comment:
should this be "all all points of interconnections" and not just " with outside networks"?

Reply:
@pgoyal01 You are right
Replaced by "all points of interconnections (internal or with outside networks)"